eng/Publishing.props

```diff
@@ -1,8 +1,8 @@
-<Project>
-
-  <ItemGroup>
-    <Artifact Include="$(ArtifactsDir)nupkgs/*.nupkg"
-              PublishFlatContainer="false" />
-  </ItemGroup>
-  
-</Project>
+<Project>
```

Comment:
You probably need to do `git checkout dev -- eng/Publishing.props` to revert this change.

I'm guessing it's BOM?
I wonder if the rest of us will get this when we try to create a PR.